libs/pipelines/api_pipeline.py

```diff
@@ -80,60 +102,118 @@ def run_projections(
 
 def _generate_api_without_ts(projection_result, row, input_dir):
     if projection_result.aggregation_level == AggregationLevel.STATE:
-        generated_data = api.generate_api_for_state_projection_row(
-            row
-        )
+        generated_data = api.generate_api_for_state_projection_row(row)
     elif projection_result.aggregation_level == AggregationLevel.COUNTY:
-        generated_data = api.generate_api_for_county_projection_row(
-            row
-        )
+        generated_data = api.generate_api_for_county_projection_row(row)
     else:
         raise ValueError("Aggregate Level not supported by api generation")
     key_prefix = _get_api_prefix(projection_result.aggregation_level, row)
     generated_key = f"{key_prefix}.{projection_result.intervention.name}"
-    return APIGenerationRow(generated_key, generated_data)
+    return APIOutput(generated_key, generated_data)
 
 
 def _generate_api_with_ts(projection_result, row, input_dir):
     if projection_result.aggregation_level == AggregationLevel.STATE:
-        generated_data = api.generate_api_for_state_timeseries(
+        generated_data = api.generate_state_timeseries(
             row, projection_result.intervention, input_dir
         )
     elif projection_result.aggregation_level == AggregationLevel.COUNTY:
-        generated_data = api.generate_api_for_county_timeseries(
+        generated_data = api.generate_county_timeseries(
             row, projection_result.intervention, input_dir
         )
     else:
         raise ValueError("Aggregate Level not supported by api generation")
     key_prefix = _get_api_prefix(projection_result.aggregation_level, row)
     generated_key = f"{key_prefix}.{projection_result.intervention.name}.timeseries"
-    return APIGenerationRow(generated_key, generated_data)
+    return APIOutput(generated_key, generated_data)
 
 
 def generate_api(
```

Comment:
I think we could refactor this to just return two lists then we wouldn't have to do that type checking stuff?

Reply:
Yeah I was just thinking that, if it returned summaries and summaries with timestamp life would be much easier